JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1427,4 +1455,89 @@ Descriptives <- function(jaspResults, dataset, options) {
     return(TRUE)
   else
     return(FALSE)
-}
\ No newline at end of file
+}
+
+.Xbarchart <- function(jaspResults, dataset, options, variables){
+
+  ready <- (length(options$variables) > 0)
+
+  if (ready && is.null(jaspResults[["Xbarchart"]]))
+    .XbarPlotDescriptives(jaspResults, dataset, options, ready)
+
+  return()
+}
+
+.XbarPlotDescriptives <- function(jaspResults, dataset, options, ready){
+
+  XbarPlot <- createJaspPlot(title = "X bar chart", width=500, aspectRatio=1)
+  jaspResults[["Xbarchart"]] <- XbarPlot
+
+  if (!ready)
+    return()
+
+  .XbarFillPlotDescriptives(XbarPlot, dataset, options)
+
+  return()
+}
+
+.XbarFillPlotDescriptives <- function(XbarPlot, dataset, options){
+
+  plot <- qcc::qcc(dataset, type="xbar", plot= F)
```

Comment:
This is a nice package, but it doesn't return ggplot2 objects so once we have plot editing these plots cannot be modified. Can you remake the plots with `ggplot2`?